### PR TITLE
Feature/board remove

### PR DESCRIPTION
**Pull request by jaenrig-ifx**
By creating this pull request you agree to the terms in CONTRIBUTING.md.
https://github.com/Infineon/.github/blob/master/CONTRIBUTING.md
--- DO NOT DELETE ANYTHING ABOVE THIS LINE ---

CONTRIBUTING.md also tells you what to expect in the PR process.

Description
Removed discontinued, unverified and unsupported boards in v2.x. Set architecture to 'xmc'.

### CODE REVIEW

**Comment by jaenrig-ifx** [comment]
To properly retrieve the manifest, I had to create a dummy "latest" release-> Once we have the first 2.x version, we can remove that. For now, that will support the ci/cd runs.

File: .github/workflows/compile-platform-examples.yml [hunk 1]
```diff
@@ -53,27 +53,17 @@ jobs:
             stackMem: true
             dma: false
             alarmRtc: false
-          - fqbn: Infineon:arm:XMC1100_H_BRIDGE2GO
+          - fqbn: Infineon:xmc:XMC4200_Platform2GO
```
Comment (techpaul) [inline .github/workflows/compile-platform-examples.yml:56]: Incorrect Board name gives TWO XMC4200 Platform 2 Go

**Review by techpaul — COMMENTED** [review]
WOW, dropping support fo a whole range of boards, Now I have to have TWO versions installed to support old and new boards.

I can understand  dopping XMC1400 Boot Kit and XMC4800 as they were never fully released.

Now I have potentially unsupported boards in a few sites, thank you.

**Review by boramonideep — APPROVED** [review]

**Review by djaumann — APPROVED** [review]

File: .github/workflows/compile-platform-examples.yml (hunk 1 above)
Comment (jaenrig-ifx) [reply .github/workflows/compile-platform-examples.yml:56]: What do you mean? there is a XMC4**2**00 and XMC4**4**00 Platform 2Go.

File: README.md [hunk 2]
```diff
@@ -31,13 +29,11 @@ Please visit also the Wiki for additional information, e.g. datasheets, pin out
 [XMC-for-Arduino Wiki](https://github.com/Infineon/XMC-for-Arduino/wiki)
 
 * Page for [XMC1100 XMC 2Go](https://github.com/Infineon/XMC-for-Arduino/wiki/XMC-2Go)
-* Page for [XMC1100 H-Bridge 2Go](https://github.com/Infineon/XMC-for-Arduino/wiki/XMC1100-H%E2%80%90Bridge-2Go)
 * Page for [XMC1100 Boot Kit](https://github.com/Infineon/XMC-for-Arduino/wiki/XMC1100-Boot-Kit)
-* Page for [XMC1300 Boot Kit](https://github.com/Infineon/XMC-for-Arduino/wiki/XMC1300-Boot-Kit)
-* Page for [XMC1300 Sense2GoL](https://github.com/Infineon/XMC-for-Arduino/wiki/XMC1300-Sense2GoL)
+* Page for [XMC1400 Arduino Kit](https://github.com/Infineon/XMC-for-Arduino/wiki/XMC1400-Kit-for-Arduino)
```
Comment (boramonideep) [inline README.md:33]: XMC1400 Kit for Arduino
```suggestion
* Page for [XMC1400 Kit for Arduino](https://github.com/Infineon/XMC-for-Arduino/wiki/XMC1400-Kit-for-Arduino)
```

**Review by boramonideep — CHANGES_REQUESTED** [review]

**Pull request merged**